src/main/app/src/app/zaken/zaken-verdelen-dialog/zaken-verdelen-dialog.component.ts

```diff
@@ -77,4 +102,22 @@ export class ZakenVerdelenDialogComponent implements OnInit {
             return null;
         }
     }
+
+    getGroep(): Groep {
```

Comment:
Ik vind dit wel een apart stukje, waarom zit het hele object niet in de control.value?